### PR TITLE
Automation tests setup

### PR DESCRIPTION
**Pull request by MilanVojnovic95**
### This PR includes

- Playwright setup ✅ 
- Install Synpress Library ✅ 
- added couple selectors on FE ✅ 
- Add Example tests for connecting Metamask to Carrop app ✅

Note: connectWallet.test.ts will work only on `local` env since selectors are not yet on `stage` env

### CODE REVIEW

File: automation-tests/tests/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+## Prerequisites
+
+-   Node version _(16.x, 18.x, 20.x\)_
+-   Playwright
+
+## Installation
+
+```
+Yarn i -y
```
Comment (guerrap) [inline automation-tests/tests/README.md:21]: This should be lowercased `yarn`.

File: automation-tests/tests/homePage.test.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+//import { test, expect, selectors } from '@playwright/test';
+import { test, expect } from "../tests/utils/fixtures";
+
+test('Carrot Home page', async ({ page }) => {
+    await test.step('Navigate to Home page', async () => {
+        await page.goto('https://app.staging.carrot.community/#/?chain=sepolia');
```
Comment (guerrap) [inline automation-tests/tests/homePage.test.ts:6]: The `goto` should  use the configured `baseUrl`, no?

File: automation-tests/tests/README.md (hunk 1 above)
Comment (MilanVojnovic95) [reply automation-tests/tests/README.md:21]: done

File: automation-tests/tests/homePage.test.ts (hunk 2 above)
Comment (MilanVojnovic95) [reply automation-tests/tests/homePage.test.ts:6]: yes, done

**Comment by luzzif** [comment]
Closing in favor of https://github.com/carrot-kpi/monorepo/pull/502

**Pull request closed without merging**